test/es-module/test-require-module-preload.js

```diff
@@ -1,70 +1,117 @@
 'use strict';
 
 require('../common');
-const { spawnSyncAndExitWithoutError } = require('../common/child_process');
-const fixtures = require('../common/fixtures');
-
+const { spawnSyncAndAssert } = require('../common/child_process');
+const { fixturesDir } = require('../common/fixtures');
 const stderr = /ExperimentalWarning: Support for loading ES Module in require/;
 
-// Test named exports.
-{
-  spawnSyncAndExitWithoutError(
-    process.execPath,
-    [ '--experimental-require-module', '-r', fixtures.path('../fixtures/es-module-loaders/module-named-exports.mjs') ],
-    {
-      stderr,
-    }
-  );
-}
+function testPreload(preloadFlag) {
+  // Test named exports.
+  {
```

Comment:
nit: no need for those blocks since we're not declaring any variable

Reply:
It’s mostly for folding in the editors that support folding blocks.

Reply:
Doesn't most editors treat multiline function calls as foldable blocks?

Reply:
In my experience it doesn't always work correctly, but blocks almost never fail.